freeipa/src/main/java/com/sequenceiq/freeipa/kerberosmgmt/v1/KerberosMgmtV1Service.java

```diff
@@ -394,7 +411,7 @@ private static String constructCanonicalPrincipal(String serviceName, String hos
 
     private static class VaultPathBuilder {
```

Comment:
this inner class could be moved out and then separately covered with unit tests, so we could ensure it's backward compatible

Reply:
Done